server/grpc_service.go

```diff
@@ -1950,7 +1950,7 @@ func (s *GrpcServer) LoadGlobalConfig(ctx context.Context, request *pdpb.LoadGlo
 	}
 	res := make([]*pdpb.GlobalConfigItem, len(r.Kvs))
 	for i, value := range r.Kvs {
-		res[i] = &pdpb.GlobalConfigItem{Kind: pdpb.EventType_PUT, Name: string(value.Key), Payload: value.Value}
+		res[i] = &pdpb.GlobalConfigItem{Kind: pdpb.EventType_PUT, Name: string(value.Key[len(configPath):]), Payload: value.Value}
```

Comment:
why not handle in client. I think you use `clientv3.WithPrefix()` that expect to get the key with the prefix.

Reply:
The purpose of the `clientv3.WithPrefix()`  is to get all the keys under this configuration path
I think it makes more sense to return keys that don't care about prefixes.
For loading configurations, the server's current implementation is to provide two parameters, Names and ConfigPath.
- Filling `Names` will literately get the values from `ConfigPath/Name`, the keys now have no prefix 
- If `Names` is empty, all values of the current`ConfigPath` will be obtained, but the keys are with prefix.
Consistency cannot be achieved

Reply:
Okay, keep the client get key with config path and just return prekv when is a delete event.